src/items/menu.js

```diff
@@ -110,7 +110,7 @@ const makeTree = (core, __, metadata) => {
 export default class MenuPanelItem extends PanelItem {
 
   init(state, actions) {
-    window.addEventListener('keydown', ev => {
+    const onkeydown = ev => {
       const checkKeys = this.core.config('panel.menu.keys', 'Alt+a').toLowerCase().split('+');
```

Comment:
This configuration namespace does not exist, and I think maybe this should be moved inside the actual settings on the desktop.

Example:

```
{
  desktop: {
    settings: {
      panels: [{
        position: 'top',
        items: [
          {name: 'menu', options: {boundKey: 'Alt+a'}}, // Static default value
          // .. all other default panel items
        ]
      }]
    }
  }
}
```

Doing this also makes it possible to configure at runtime since these options can be modified and stored into [user] `Settings` automatically.

But looking at the panel item code, the `options` are currently not passed on to the constructor, but I will fix that ASAP.

So after that has been implemented, this would instead be:

```
const checkKeys = this.options.boundKey || 'Alt+a';
```

Reply:
FYI, you can still change it to the snippet above, it will still work.